src/Facility.CodeGen.CSharp/CSharpGenerator.cs

```diff
@@ -32,6 +32,11 @@ public sealed class CSharpGenerator : CodeGenerator
 		/// </summary>
 		public bool UseNullableReferences { get; set; }
 
+		/// <summary>
+		/// The serializers to generate support for. Defaults to all.
+		/// </summary>
+		public IReadOnlyCollection<ServiceSerializerKind> Serializers { get; set; } = s_defaultSerializers;
```

Comment:
Should this really be a setting? I can imagine a future where I don't want to depend on Newtonsoft.Json, or where there's a Protobuf serializer, but maybe that should just be saved for later.

Reply:
I don't think we need it.